tools/shared/utils/symutil.cpp

```diff
@@ -700,7 +700,9 @@ class Symutil : public UtilityApplication
 
     // write dinit for STB
     out3 << "\nextern STB stb;\n";  // declaration into .h file
-    out4 << "\n#include \"symacc.h\"\n"; // forward declaration of the STB type.
+    out4 << "\n#include \"symacc.h\"\n"; // forward declaration of the STB type
+    out4 << "\n#pragma clang diagnostic ignored ";
```

Comment:
A word of warning that this line is continued below would help, it took me a while to figure out what's going on here...

Reply:
Fixed. Hopefully it is more understandable now.